### PR TITLE
PrasadC_Assignment12

### PR DESCRIPTION
**Pull request by PrasadCOct2020**
Hi Both,

Please review my PR. PFB screenshot of output.
image

Thanks
Prasad Chitale

### CODE REVIEW

**Comment by PrasadCOct2020** [comment]
Output after adding reverse order method.
image

File: src/prasad/Array.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	}
+	
+	public static void main (String[]arg) {
+		int [] array = new int [5];
+		Array arrayvar = new Array();
+		arrayvar.setData(array);
+		arrayvar.display(array);
+		double avg = arrayvar.arrayavg(array);
+		System.out.println("Average of array: " + avg);
+		int[] array1 = {9,10,-1,80,12};
```
Comment (harshvegada) [inline src/prasad/Array.java:63]: if you are passing data from here then  what is use of setData method?

File: src/prasad/Array.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	double arrayavg(int[]num) {
+		double avg=0;
+		float sum = arraysum(num);
+		System.out.println("Sum of array: " + sum);
+		avg = (sum/num.length);
+		return avg;
+	}
+	float max (int[]num) {
+		int max= num[0];
+		for(int index=0;index<num.length;index++) {
```
Comment (harshvegada) [inline src/prasad/Array.java:37]: Video send for this code on skype

**Review by harshvegada — APPROVED** [review]
Hi @PrasadCOct2020 ,
Very well work,
this is something i am expecting.
Keep it up!

**Pull request merged**